doctrine/mongodb-odm-bundle/3.3/etc/packages/doctrine_mongodb.yaml

```diff
@@ -0,0 +1,18 @@
+doctrine_mongodb:
+    auto_generate_proxy_classes: "%kernel.debug%"
+    auto_generate_hydrator_classes: "%kernel.debug%"
+    connections:
+        default:
+            server: "%env(MONGODB_URL)%"
+            options: {}
+    default_database: "%env(MONGODB_DBNAME)%"
+    document_managers:
+        default:
+            auto_mapping: true
+            mappings:
+                App:
+                    is_bundle: false
+                    type: annotation
+                    dir: "%kernel.project_dir%/src/Document"
+                    prefix: "App\\Document\\"
```

Comment:
We often voluntarily remove quotes for class names and would just use `prefix: App\Document\` in this case, it should work